In [7]:
mu <- .8893
# Major => 1 = Econ, 2 = Anthro, 3 = Sociology
a1 <- 0
a2 <- 1.9860
a3 <- 1.1889
# High School Back Ground (BG) => 1 = Rural, 2 = Urban
eta1 <- 0
eta2 <- 1.2564
# Interaction
gam11 <- 0
gam12 <- 0
gam21 <- 0
gam22 <- -1.6631
gam31 <- 0
gam32 <- -1.6130


#                                   Estimate  Std. Error  t value  Pr(>|t|)
# (Intercept)                         0.8893      0.4033    2.205   0.03554 *
# as.factor(Major)2                   1.9860      0.6377    3.114   0.00413 **
# as.factor(Major)3                   1.1889      0.6377    1.864   0.07244 .
# as.factor(BG)2                      1.2564      0.5207    2.413   0.02237 *
# as.factor(Major)2:as.factor(BG)2   -1.6631      0.8233   -2.020   0.05270 .
# as.factor(Major)3:as.factor(BG)2   -1.6130      0.8233   -1.959   0.05977 .

In [14]:
(E_y11 <- mu + a1 + eta1 + gam11) # = mu
(E_y12 <- mu + a1 + eta2 + gam12) # = mu + eta2
(E_y21 <- mu + a2 + eta1 + gam21) # = mu + a2
(E_y22 <- mu + a2 + eta2 + gam22) #
(E_y31 <- mu + a3 + eta1 + gam31) # = mu + a3
(E_y32 <- mu + a3 + eta2 + gam32) # = mu + a3


[1] 0.8893

[1] 2.1457

[1] 2.8753

[1] 2.4686

[1] 2.0782

[1] 1.7216

### 1a) Which group of students has the lowest score?

Note that 

$$
\begin{array}{rclclcr}
    E[y_{11}] &=& \mu + \alpha_1 + \eta_1 + \gamma_{11} &\approx& \hat\mu &=& .8893 \\
    E[y_{12}] &=& \mu + \alpha_1 + \eta_2 + \gamma_{12} &\approx& \hat\mu +\hat\eta_2 &=& 2.1457 \\
    E[y_{21}] &=& \mu + \alpha_2 + \eta_1 + \gamma_{21} &\approx& \hat\mu +\hat\alpha_2 &=& 2.8753 \\
    E[y_{22}] &=& \mu + \alpha_2 + \eta_2 + \gamma_{22} &\approx& \hat\mu + \hat\alpha_2 + \hat\eta_2 + \hat\gamma_{22}&=& 2.4686 \\
    E[y_{31}] &=& \mu + \alpha_3 + \eta_1 + \gamma_{31} &\approx& \hat\mu +\hat\alpha_3 &=& 2.0782 \\
    E[y_{32}] &=& \mu + \alpha_3 + \eta_2 + \gamma_{32} &\approx& \hat\mu + \hat\alpha_3 + \hat\eta_2 + \hat\gamma_{32}&=& 1.7216 \\
\end{array}
$$


So, the group that corresponds to $y_{11}$ has the **lowest** math ineptitude score. That is, **Economics** majors from **rural** high school backgrounds, with a score of **.8893**.

The group that corresponds to $y_{21}$ has the **highest** math ineptitude score. That is, **Anthropology** majors from **rural** high school backgrounds, with a score of **2.8753**.